### PR TITLE
Remove unused AccessForbidden

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/description.md [hunk 1]
```diff
@@ -34,9 +34,6 @@ Never share your secret keys. Keep them guarded and secure.
 # Errors
 Rebilly follow's the error response format proposed in [RFC 7807](https://tools.ietf.org/html/rfc7807) also known as Problem Details for HTTP APIs.  As with our normal API responses, your client must be prepared to gracefully handle additional members of the response.
 
-## AccessForbidden
```
Comment (slavcodev) [inline openapi/description.md:37]: Should not be replaced with `Forbidden` ?

**Comment by slavcodev** [comment]
`Forbidden` is not shown, seems path is incorrect

**Comment by XaosSintez** [comment]
@slavcodev Yes, but i don't understand it yet. It is not different from `Conflict` at first glance.

**Comment by slavcodev** [comment]
@XaosSintez most probably it's unused and redocly does not build it in final json.

**Comment by XaosSintez** [comment]
It is used in many places. For example here:
https://github.com/Rebilly/api-definitions/blob/09e79aa7477e4e63cffd75df1effbf6fab0231c8/openapi/paths/gateway-accounts.yaml#L38

**Comment by slavcodev** [comment]
This path is not included in core https://preview.redoc.ly/rebilly-core/remove-AccessForbidden/tag/Transactions

**Comment by XaosSintez** [comment]
@slavcodev Yes, it looks unused in core as i see. So, no reason to add it to `description.md` then?

**Comment by slavcodev** [comment]
Depends :) `description.md` is shared as far as I understand. I think we need to add `403` response in core to where is appropriate.

**Comment by XaosSintez** [comment]
As i understand, it is mostly used in ACL checks (with few uses in some endpoints). ACL check are applicable to all the endpoints. So, it can be added to almost everywhere IMO.

In this case i think makes sense to add it to single endpoint here (to make `description.md` working as expected) and create separated ticket/PR to add this response to all endpoints. 

@slavcodev What do you think?

**Review by slavcodev — APPROVED** [review]

**Comment by XaosSintez** [comment]
@slavcodev Thanks

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**